packages/vendure-plugin-frequently-bought-together/src/frequently-bought-together.plugin.ts

```
@@ -0,0 +1,67 @@
+  //   },
+})
+export class FrequentlyBoughtTogetherPlugin {
+  static options: PluginInitOptions = {
+    maxRelatedProducts: 5,
+    customFieldUiTab: 'Related products',
+  };
+
+  static init(
+    options: Partial<PluginInitOptions>
+  ): Type<FrequentlyBoughtTogetherPlugin> {
+    this.options = {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)